package.json

```diff
@@ -19,6 +19,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.11.6",
+    "@babel/plugin-transform-runtime": "^7.25.7",
```

Comment:
Some context: i took over this library about 6months ago, contributed quite a bit to it, then had to take a break as a started a new job.

I'm confused by this change because when I look at the rollup config, babelrc is set to false:
https://github.com/RickyMarou/jest-webextension-mock/blob/main/rollup.config.js#L10

After reading up on this a bit, this seems to be a configuration option to prevent dependencies from using their own babelrc files, and use the one from this package instead. (see [here](https://www.npmjs.com/package/@rollup/plugin-babel#external-dependencies))

This plugin is indeed present in the babelrc file, and not in the package.json. I assume adding it is related to the usage of the nullish coalescing operator (`??`) in the alarm.js file?

I tried removing this package and removing `@babel/plugin-transform-runtime` and the extension is still building fine, peaking at the code in the bundle, the alarm.js code seems to be transpiled correctly.

I suspect that adding this package is superfluous, but maybe you have a different understanding of the build toolchain here?

Reply:
Sorry for long no reply. Yes, this is in fact needless. I removed it.